site/content/en/docs/contrib/roadmap.en.md

```diff
@@ -6,7 +6,7 @@ description: >
   Development Roadmap
 ---
 
-This roadmap is a living document outlining the major technical improvements which we would like to see in minikube during 2020, divided by how they apply to our [guiding principles](principles.en.md)
+This roadmap is a living document outlining the major technical improvements which we would like to see in minikube during 2020, divided by how they apply to our [guiding principles](https://minikube.sigs.k8s.io/docs/contrib/principles/)
```

Comment:
please use "Ref" for internal links, so it can be loaded on local sites when testing on personal machine
for example
```
* [Addons]({{< ref "/docs/handbook/deploying.md#addons" >}})
```

Reply:
Incorporated the comments